src/cc/lib/py_graph.cc

```diff
@@ -621,6 +621,41 @@ int32_t GetNeighborsInternal(PyGraph *py_graph, NodeID *in_node_ids, size_t in_n
     return 0;
 }
 
+int32_t NeighborCount(PyGraph *py_graph, NodeID *in_node_ids, size_t in_node_ids_size, Type *in_edge_types,
+                      size_t in_edge_types_size, uint64_t *out_neighbor_counts)
+{
+    if (py_graph->graph == nullptr)
+    {
+        RAW_LOG_ERROR("Internal graph is not initialized");
+        return 1;
+    }
+    std::fill_n(out_neighbor_counts, in_node_ids_size, 0);
```

Comment:
Why do you need to perform `std::_fill` twice? It is done first time here and second time in `graph.cc`/`client.cc`.